src/pages/registration/DescriptionPanel.tsx

```diff
@@ -115,6 +115,11 @@ const DescriptionPanel: FC<PanelProps> = ({ setTouchedFields }) => {
                     helperText={t('description.keywords_helper')}
                     variant="outlined"
                     fullWidth
+                    onBlur={(event: any) => {
```

Comment:
får du ikke riktig type out-of-the-box hvis du bare ikke angir type?
```suggestion
                    onBlur={(event) => {
```

Reply:
jo 🙃  #fredag